### DISCUSSION TITLE
Feature Request: Option to disable tabs completely

### DISCUSSION THREAD

**Discussion by minthemiddle** [notaapp/talk#364] [Feature Requests]
The new tab and link settings in `0.39.x` don't reduce tab amounts for me at all.
I still have way too many open all the times and have no advantage of them at all.

I would strongly prefer to see no tabs instead and just use the command palette (cmd + p) to go to my last edited files.
Of course, this is not for everyone, that's why maybe an optional setting would be best.

**Comment by astoilkov** [comment]
I agree. We didn't make good improvements in reducing the number of tabs in 0.39.  However, we're confident we have a solution for 0.40 that will fix the problem. It's something that we struggle with as well.

**Comment by Auratelience** [comment]
I did like the button tab icons.

**Comment by greenieves** [comment]
Same. Would love to disable tabs entirely or at least hide the tab bar.
